website/docs/reference/change-requests.md

```diff
@@ -62,7 +63,43 @@ From here, you can navigate to the change request overview page. This page will
 
 ![Change request banner](/img/change-request-overview.png)
 
-From here, if you have the correct permissions, you can approve and apply the change request. Once applied the changes will be live in production.
+From here, if you have the correct permissions, you can approve and schedule or apply the change request. Once applied, the changes will be live in production.
+
+### Scheduled changes
+
+:::info Availability
+
+Change request scheduling is currently in development and will be released in an upcoming version of Unleash. How the feature works (and as such, the contents of this subsection) can change before the feature is released.
+
+:::
+
+When a change request is approved, you can schedule it to be applied at a later time. This allows you to group changes together and apply them at a time that is convenient for you, such as during a maintenance window, or at a time when you know there will be less traffic to your application.
+
+Scheduled changes can be rescheduled, applied immediately, or rejected. They can not be edited or moved back to any of the previous states.
```

Comment:
I think that atm you can edit something in a scheduled request.  This would move it back in state

Reply:
I'm pretty sure you can't. There's tests checking for that explicitly in the enterprise version. Once it's scheduled, it can only move into applied, rejected, or scheduled. Check `src/change-request/change-request.test.ts`.